pkg/uploader/types.go

```diff
@@ -49,14 +49,16 @@ func ValidateUploaderType(t string) (string, error) {
 }
 
 type SnapshotInfo struct {
-	ID   string `json:"id"`
-	Size int64  `json:"Size"`
+	ID              string `json:"id"`
+	Size            int64  `json:"Size"`
+	IncrementalSize int64  `json:"IncrementalSize"`
 }
 
 // Progress which defined two variables to record progress
 type Progress struct {
-	TotalBytes int64 `json:"totalBytes,omitempty"`
-	BytesDone  int64 `json:"doneBytes,omitempty"`
+	TotalBytes       int64 `json:"totalBytes,omitempty"`
+	BytesDone        int64 `json:"doneBytes,omitempty"`
+	IncrementalBytes int64 `json:"incrementalBytes,omitempty"`
```

Comment:
See https://github.com/vmware-tanzu/velero/pull/9357/files#r2450263424

Reply:
@sseago 
I see a inconsistent of your two comments above. Anyway, this should be removed yet, I don't see anywhere is using this field.